text-ui-test/input.txt

```diff
@@ -0,0 +1,13 @@
+batman
+todo make batarangs
+deadline stop joker's escape from arkham asylum /by tonight
+list
+mark 1
+list
+event charity fundraising gala dinner /from sunday 5pm /to sunday 11pm
```

Comment:
Bad move by batman, this could compromise his secret identity if the villains saw this task under his username in artemis 😄